packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -312,20 +312,18 @@ export class FakeChannelProvider implements ChannelProviderInterface {
     return {success: true};
   }
 
-  private async approveBudgetAndFund(params: BudgetRequest): Promise<SiteBudget> {
-    const {hub, site, playerAmount, hubAmount} = params;
-
+  private async approveBudgetAndFund(params: TokenBudgetRequest): Promise<SiteBudget> {
     // TODO: Does this need to be delayed?
-    const result = {
-      hub: hub.signingAddress,
-      site,
+    const result: SiteBudget = {
+      hubAddress: params.hub.signingAddress,
+      domain: 'web3torrent.statechannels.org',
```

Comment:
Should we have a reference to web3torrent baked into the fake channel provider? Other apps such as RPS might (do) use it.

Reply:
ecd818d